_pages/develop/map.md

```diff
@@ -0,0 +1,682 @@
+Future iterations of the component may offer the ability to use a different
+library in place of Leaflet, while keeping the same basic API.
+
+Out of the box, `px-map` includes the basic things needed to draw and plot data
+on an interactive map. It is also extensible — development teams are encouraged
+to build their own subcomponents that add features needed for their application.
+We currently do not support visualizations, but any open source Leaflet plugin
+can be easily converted to be a px-map custom layer.
+
+# Basic concepts
+
+Use the map to plot geographic data such as individual assets or clusters of
+assets and information about them. The data concepts underlying the map include:
+
+- **Base layer:** The base layer of a map renders tile images that include
+information such as streets, mountains, and national parks. For example,
+satellite, terrain, and map view are all different base layer styles.
+- **Overlay:** Information that is put on top of a map's base layer that is
+specific to the purpose of that map. For example, traffic data or bike lane
+information might be set as an overlay on a road map.
+- **Control:** A map layer that allows the user to manipulate the map, e.g. by
+zooming in, out, or to their current location.
+- **CRS:** A CRS is a coordinate reference system (also known as a coordinate
+relationship system or coordinate projection system) used to project
+geographical points into pixel coordinates.
+
+## Graphic: map layers
+
+# Subcomponents
+
+The following subcomponents are available for use with px-map:
+
+**Base map**
+
+- `<px-map>`: Draws the underlying map, sets location/zoom and notifies updates
+when the user interacts with the map, enables and disables interactions,
+orchestrates subcomponents
+
+**Tile layers**
+
+- `<px-map-tile-layer>`: Calls a tile service API to fetch underlying tile images
+for the map
+- `<px-map-tile-layer-bing>`: Calls the Bing Maps API to fetch underlying tile
+images for the map
+- `<px-map-tile-layer-google>`: Calls the Google Maps API to fetch underlying
+tile images for the map
+- `<px-map-tile-layer-wms>`: Displays WMS services as tile layers on the map.
+
+**Overlay/visualization layers**
+
+- `<px-map-layer-geojson>`: Draws GeoJSON data as vectors on the map
+- `<px-map-layer-group>`: Groups related overlays together to allow for bulk
+interactions (e.g. hide all in the group)
+- `<px-map-marker-static>`: Creates a marker that shows the state of a point/asset
+(e.g. with color)
+- `<px-map-marker-symbol>`: Creates a marker with an icon that shows the state
+of a point/asset
+- `<px-map-marker-locate>`: Creates a marker that represents the user's location
+- `<px-map-marker-group>`: Draws many markers in clusters, useful for visualizing
+thousands of points that can be dynamically updated
+
+**UI components**
+
+- `<px-map-popup-info>`: Binds a popup that can include text or an image to a marker
```

Comment:
Inconsistent use of "Adds" "Creates" "Binds" and "Draws" throughout this section, maybe consider standardizing or omitting the verb if it isn't conveying any specific meaning?

Reply:
I standardized the verbs for each section (e.g. "draws" for all the overlay layers), but I think we're using different verbs for a reason (e.g. "draws" vs "binds") so I'm going to keep it like that